src/main/java/com/sap/sgs/phosphor/fosstars/data/github/UsesSignedCommits.java

```diff
@@ -0,0 +1,209 @@
+          break;
+        }
+        pageNumber++;
+
+        for (JsonNode commitNode : commitNodeList) {
+          counter++;
+          JsonNode verifiedNode = commitNode.findPath("verified");
+          if (verifiedNode != null && verifiedNode.booleanValue()) {
+            signedCounter++;
+          }
+        }
+      }
+    } catch (IOException e) {
+      System.out.printf("[x] Couldn't fetch data from api.github.com!%n");
+      System.out.printf("[x]     %s%n", e);
+      return UnknownValue.of(USES_VERIFIED_SIGNED_COMMITS);
+    }
+
+    if (counter > 0 && signedCounter > 0
+        && (signedCounter * 100 / counter) >= SIGNED_COMMIT_PERCENTAGE_THRESHOLD) {
+      return new BooleanValue(USES_VERIFIED_SIGNED_COMMITS, true);
+    }
+    return new BooleanValue(USES_VERIFIED_SIGNED_COMMITS, false);
+  }
+
+  /**
+   * Does a REST API call to public URL <a link="https://api.github.com">GitHub API</a> to list all
+   * the commits after a specific date.
+   * 
+   * @param date Only commits after this date will be returned. 
+   *             This is a timestamp in ISO 8601, format: YYYY-MM-DDTHH:MM:SSZ
+   * @param page Index of the page
+   * @return The {@link JsonNode} object
+   * @throws IOException #{@link HttpClient} may throw an exception during REST call.
+   * @see: GitHub Issue: <a link="https://github.com/github-api/github-api/issues/737">Get commit or
+   *       tag signature verified flag</a> and the associated
+   *       <a link= "https://github.com/github-api/github-api/pull/738">Pull Request</a>.
+   */
+  private JsonNode commitInfo(String date, int page) throws IOException {
+    // TODO: Remove this method when org.kohsuke:github-api implements GHCommit::verification. More
+    // information can be found in https://github.com/github-api/github-api/issues/737
+    CloseableHttpClient client = httpClient();
+    String url =
+        String.format("https://api.github.com/repos/%s/%s/commits?since=%s&page=%s&per_page=%s",
+            where, name, date, page, MAX_ITEMS_PER_PAGE);
+
+    HttpGet httpGetRequest = new HttpGet(url);
+    httpGetRequest.addHeader(HttpHeaders.ACCEPT, ContentType.APPLICATION_JSON.getMimeType());
+    if (githubToken != null && !githubToken.isEmpty()) {
+      httpGetRequest.addHeader(HttpHeaders.AUTHORIZATION, String.format("token %s", githubToken));
+    }
+
+    CloseableHttpResponse httpResponse = client.execute(httpGetRequest);
+    
+    // Check if next page is available.
+    hasNextPage = hasNextPage(httpResponse);
+    return MAPPER.readTree(httpResponse.getEntity().getContent());
+  }
+
+  /**
+   * Returns an HTTP client.
+   */
+  CloseableHttpClient httpClient() {
+    return HttpClients.createDefault();
+  }
+
+  /**
+   * Check if next page is available, returns true. Otherwise the last page has been reached.
+   * 
+   * @param response {@link CloseableHttpResponse} from the GET Request.
+   * @return boolean if the header "link" contains 'rel="next"'
+   */
```

Comment:
Would you mind updating this to the following:

```
   * Check if the next page is available.
   * 
   * @param response {@link CloseableHttpResponse} from the GET Request.
   * @return True if the next page is available, false otherwise.
```